src/test/java/org/fundacionjala/coding/christian/Kata1Test.java

```diff
@@ -0,0 +1,37 @@
+package org.fundacionjala.coding.christian;
+
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+/**
+ * Powered by Christian.
+ */
+public class Kata1Test {
+    /**
+     *
+     */
+    @Test
+    public void testAllWordsCantBeReversed() {
+        Kata1 words = new Kata1();
```

Comment:
You are repeating this line of code in each test.
Consider using a @Before method.